bgpd/bgp_attr.c

```diff
@@ -43,6 +43,9 @@
 #include "bgp_linkstate_tlv.h"
 #include "bgp_mac.h"
```

Comment:
I would change the commit log to mention that I declare a new attribute.: BGP_ATTR_LS.

Reply:
I have added `New types of data BGP_ATTR_LS and BGP_ATTR_LS_DATA are defined.`